metrics/src/macros.rs

```diff
@@ -86,13 +90,17 @@ macro_rules! gauge {
 
 /// Records a timing.
 ///
+/// This will register an histogram with the given name, if it does not already
+/// exist, then add data point with the given timing. This timing must implement
+/// `AsNanoseconds`. Optionally, a set of labels, of the form `key => value`,
```

Comment:
You have to wrap the type in brackets -- ``[`AsNanoseconds`]`` -- for it to become a hyperlink.

Reply:
a linting tool for dead links would be a godsend.